org.civicrm.reqangular/js/test/test-main.js

```diff
@@ -1,10 +1,10 @@
 /* globals CRM */
 
-var TEST_REGEXP = /(spec|test)\.js$/i;
+var TEST_REGEXP = /spec\.js$/i;
```

Comment:
May be  `/\.spec\.js$/i` would be safer to avoid matching files like `autospec.js`